ci/build_python.sh

```diff
@@ -3,14 +3,12 @@
 
 set -euo pipefail
 
-rapids-configure-conda-channels
+UPLOAD_PACKAGES="${1:-false}"
 
-source rapids-date-string
+PKG_DIR="${PWD}/dist/"
```

Comment:
It feels weird to put conda packages in a `dist` folder. That's usually what you do with wheels. Maybe `conda_packages` or `build_artifacts` or something like that?

Reply:
Sure.